### PR TITLE
Misc minor fixes & improvements

### PR DESCRIPTION
**Pull request by SergeyPiskunov**
- making the code a little bit more `pythonic`
- removing unused objects, redundant calls, fixing typos, pep8

### CODE REVIEW

File: ox_profile/core/recording.py [hunk 1]
```diff
@@ -1,33 +1,13 @@
...
-        "Return string reprsentation."
-        result = '%s(%s=%s, %s=%s)' % (
-            self.__class__.__name__, 'name', self.name, 'hits', self.hits)
-
-        return result
-
-    def __repr__(self):
-        return self.to_str()
+# Simple record to track how many times a function/path is called.
+ProfileRecord = namedtuple('ProfileRecord', 'name, hits')
```
Comment (emin63) [inline ox_profile/core/recording.py]: I like namedtuple as well but want to be able to update fields in ProfileRecord so prefer to keep it as a class not tuple. Please revert those changes.

File: ox_profile/core/metrics.py [hunk 2]
```diff
@@ -51,11 +51,9 @@ def snap(self, frame):
 
         """
         stack = []
-        self.name = '{}({})'.format(frame.f_code.co_name,
-                                    frame.f_globals.get('__name__'))
+        self.name = '%s(%s)' % (frame.f_code.co_name, frame.f_globals.get('__name__'))
```
Comment (emin63) [inline ox_profile/core/metrics.py]: I'm OK with changing string formatting but would like to keep to the convention that source code lines are less than 80 characters. Please fix formatting to match that python convention.

File: ox_profile/core/metrics.py [hunk 3]
```diff
@@ -51,11 +51,9 @@ def snap(self, frame):
...
         while frame is not None:
-            formatted_frame = '{}({})'.format(frame.f_code.co_name,
-                                              frame.f_globals.get('__name__'))
+            formatted_frame = '%s(%s)' % (frame.f_code.co_name, frame.f_globals.get('__name__'))
```
Comment (emin63) [inline ox_profile/core/metrics.py]: Same as previous comment: please fix to be less than 80 characters wide.

File: ox_profile/core/recording.py [hunk 4]
```diff
@@ -94,11 +73,9 @@ def query(self, re_filter='.*', max_records=10):
                 name_list = name.split(';')
                 for fname in name_list:
                     if regexp.search(fname):
-                        calls[fname] = calls.get(fname, 0) + item
-            my_hits = list(reversed(sorted(calls.items(),
-                                           key=lambda pair: pair[1])))
-            result = [ProfileRecord(name, hits) for name, hits in my_hits[
-                :max_records]]
+                        calls_counter[fname] += item
+            my_hits = calls_counter.most_common(max_records)
```
Comment (emin63) [inline ox_profile/core/recording.py:104]: Hmm, nice. I didn't know about that feature of counter.

**Review by emin63 — CHANGES_REQUESTED** [review]
Looks great. Thanks for the fixes and suggestions.

If you could fix the formatting issues which cause lines wider than 80 characters and revert the change to namedtuple mentioned in the in-line review comments, that would be great.

Thanks again for the help.

File: ox_profile/core/metrics.py (hunk 2 above)
Comment (SergeyPiskunov) [reply ox_profile/core/metrics.py]: Ok. I made those lines less than 80 characters length again. My intention to get rid of the .format() was that it is a bit slower than %. (as far as I know)

File: ox_profile/core/metrics.py (hunk 3 above)
Comment (SergeyPiskunov) [reply ox_profile/core/metrics.py]: Fixed

File: ox_profile/core/recording.py (hunk 1 above)
Comment (SergeyPiskunov) [reply ox_profile/core/recording.py]: Ok, I've reverted that class back. I was trying to remove unnecessary overhead of creating such a simple object. Also I'm not sure why do we need that dedicated "to_str()" method... But if you are going to add some functionality to the "ProfileRecord" later - I'm ok with that.

**Review by emin63 — APPROVED** [review]
Looks great.

Thanks!

**Pull request merged**